_includes/getting-started.html

```diff
@@ -88,10 +88,12 @@ <h3>Kubernetes User:</h3>
             <!-- Kubernetes - mesheryctl -->
             <h4 style="font-size: 1.5em;">Using mesheryctl</h4>
             <div id="start-helm-v3" class="highlight-code">
-              <p class="code-box" style="height: 40px; line-height: 30px;"><code style="position: absolute;">mesheryctl system start</code></p>
+              <p class="code-box" style="height: 40px; line-height: 30px;"><code style="position: absolute;">mesheryctl system context create k8s -p kubernetes -s
+                mesheryctl system start</code></p>
```

Comment:
![image](https://user-images.githubusercontent.com/59508546/119210892-8b69e600-bacc-11eb-83cf-267bc5043ab4.png)

IG the text box can be of big size.... Let me try....

Reply:
@leecalcote I'll try opening a PR to the same branch via my fork coz it's not accepting the origin on mine

Reply:
Seems I can't edit the changes here in mine.... I'll add suggested changes then

Reply:
Set height to 70px and then we're good to go!

Reply:
@alphaX86 thanks much for this. Great catch.

By the way, if you ever feel like suggesting the change via CTRL+G (or CMD+G) in your reviews, please don't hesitate. It'll be good to collaboratively editing a PR with you.